docs/qa/CometBFT-QA-38.md

```diff
@@ -359,8 +359,198 @@ number of transactions processed per minute as compared to the baseline.
+
+
+| Experiment   | CPU                                                   |
+| ------------ | ----------------------------------------------------- |
+| **baseline** | ![cpu-avg](img38/voteExtensions/baseline_avg_cpu.png) |
+| **2k**       | ![cpu-avg](img38/voteExtensions/02k_avg_cpu.png)      |
+| **4k**       | ![cpu-avg](img38/voteExtensions/04k_avg_cpu.png)      |
+| **8k**       | ![cpu-avg](img38/voteExtensions/08k_avg_cpu.png)      |
+| **16k**      | ![cpu-avg](img38/voteExtensions/16k_avg_cpu.png)      |
+| **32k**      | ![cpu-avg](img38/voteExtensions/32k_avg_cpu.png)      |
+
+### Resident Memory
+
+The same conclusion reached for CPU usage may be drawn for the memory.
+That is, that a backlog of work is formed during the tests and catching up (freeing of memory) happens after the test is done.
+
+A more worrying trend is that the bottom of the memory usage seems to increase in between runs.
+We have investigated this in longer runs and confirmed that there is no such a trend.
+
+
+
+| Experiment   | Resident Set Size                                        |
+| ------------ | -------------------------------------------------------- |
+| **baseline** | ![rss-avg](img38/voteExtensions/baseline_avg_memory.png) |
+| **2k**       | ![rss-avg](img38/voteExtensions/02k_avg_memory.png)      |
+| **4k**       | ![rss-avg](img38/voteExtensions/04k_avg_memory.png)      |
+| **8k**       | ![rss-avg](img38/voteExtensions/08k_avg_memory.png)      |
+| **16k**      | ![rss-avg](img38/voteExtensions/16k_avg_memory.png)      |
+| **32k**      | ![rss-avg](img38/voteExtensions/32k_avg_memory.png)      |
+
+### Mempool size
+
+This metric shows how many transactions are outstanding in the nodes' mempools.
+Observe that in all runs, the average number of transactions in the mempool quickly drops to near zero between runs.
+
+
+| Experiment   | Resident Set Size                                                  |
+| ------------ | ------------------------------------------------------------------ |
+| **baseline** | ![mempool-avg](img38/voteExtensions/baseline_avg_mempool_size.png) |
+| **2k**       | ![mempool-avg](img38/voteExtensions/02k_avg_mempool_size.png)      |
+| **4k**       | ![mempool-avg](img38/voteExtensions/04k_avg_mempool_size.png)      |
+| **8k**       | ![mempool-avg](img38/voteExtensions/08k_avg_mempool_size.png)      |
+| **16k**      | ![mempool-avg](img38/voteExtensions/16k_avg_mempool_size.png)      |
+| **32k**      | ![mempool-avg](img38/voteExtensions/32k_avg_mempool_size.png)      |
+
+
+
+
+
+### Results
+
+| Scenario | Date       | Version                                                                               | Result |
+| -------- | ---------- | ------------------------------------------------------------------------------------- | ------ |
+| VESize | 2023-05-23 | v0.38.0-alpha.2 + varying vote extensions  (9fc711b6514f99b2dc0864fc703cb81214f01783) |        |
```

Comment:
```suggestion
| VESize   | 2023-05-23 | v0.38.0-alpha.2 + varying vote extensions  (9fc711b6514f99b2dc0864fc703cb81214f01783) |  PASS  |
```

I guess you have earned the right to fill that in with "pass" :-)

Reply:
Sorry, after offline discussion with @lasarojc I realise, it should be "N/A" because this is the baseline for future releases (there's nothing in the past to compare this to)